### PR TITLE
changes in helm templates

### PR DESCRIPTION
**Pull request by monotosh-avi**
added ControllerIP  in configs in values.yaml. 
let me know if it's better to keep it in configs, or under any other tag

### CODE REVIEW

**Review by sudswasavi — COMMENTED** [review]
@monotosh-avi would you do this change in 1.2.1 as well for the controllerIP?

File: helm/ako/values.yaml [hunk 1]
```diff
@@ -51,6 +51,8 @@ ControllerSettings:
   controllerVersion: "18.2.10" # The controller API version
   cloudName: "Default-Cloud" # The configured cloud name on the Avi controller.
 
+configs:
```
Comment (sudswasavi) [inline helm/ako/values.yaml:54]: If we move this inside the controllerSettings and modify our helm install instructions, do you see an issue? @monotosh-avi

Comment (monotosh-avi) [reply helm/ako/values.yaml]: moved

**Review by sudswasavi — APPROVED** [review]

**Comment by apalsule** [comment]
@monotosh-avi Please make sure FTs dont break if you cherry-pick this change to 1.2.1

**Pull request merged**